tests/easier68k/core/opcodes/test_branches.py

```diff
@@ -65,7 +66,24 @@ def test_bra_disassemble():
     assert b.size == OpSize.BYTE
     #assert b.address == who knows
 
-# test validity (for valid and invalid invluding odd instructions)
+def test_bra_validation():
+    """
+    Test if BRA is_valid commands work as expected
+    :return:
+    """
+
+    # BRA.is_valid("BRA", "$1000")
+    #    .L, .W, .B, (odd), (even)
+    pass
+
+def test_bra_length():
+    """
+    Test BRA.get_word_length()
+    :return:
+    """
+
+    # BRA.getWordLength() (.b .w .l)
```

Comment:
We should also use these tests?

Reply:
again, intentional. we are using that test.